e2e-tests/quote/cypress/e2e/journeys/quote/tell-us-about-your-policy/tell-us-about-your-single-policy-high-risk-country-high-cover-percentage.spec.js

```diff
@@ -0,0 +1,84 @@
+import { field as fieldSelector, actions } from '../../../../../../pages/shared';
+import { PAGES } from '../../../../../../content-strings';
+import { ROUTES, FIELD_IDS } from '../../../../../../constants';
+import { GBP } from '../../../../../../fixtures/currencies';
+import { COUNTRY_QUOTE_SUPPORT } from '../../../../../../fixtures/countries';
+
+const {
+  TALK_TO_AN_EXPORT_FINANCE_MANAGER_HIGH_RISK_HIGH_COVER_EXIT: { INTRO, CONTACT_EFM },
+} = PAGES;
+
+const {
+  ELIGIBILITY: { CURRENCY, CONTRACT_VALUE, PERCENTAGE_OF_COVER },
+  POLICY_LENGTH,
+} = FIELD_IDS;
+
+const {
+  QUOTE: { TELL_US_ABOUT_YOUR_POLICY, TALK_TO_AN_EXPORT_FINANCE_MANAGER_EXIT },
+} = ROUTES;
+
+const { HIGH_RISK_COUNTRY_1 } = COUNTRY_QUOTE_SUPPORT;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(
+  'Submit the quote form with high cover percentage of 95% for a high risk country and should be redirected to talk to an export finance manager page',
```

Comment:
```suggestion
  'Submit the Tell us about your single policy form with high cover percentage of 95% for a high risk country and should be redirected to talk to an export finance manager page',
```

Same in the other new "single" E2E test